### DISCUSSION TITLE
time map not working on custom object

### DISCUSSION THREAD

**Discussion by phillbarr** [deejay-hub/timeline-lwc#235] [Q&A] [answered]
![Screenshot 2023-01-26 at 16 20 08](https://user-images.githubusercontent.com/89133704/214891465-d2ed0034-93a4-40c3-83e3-a6ebcc68f6d9.png)
![Screenshot 2023-01-26 at 16 19 38](https://user-images.githubusercontent.com/89133704/214891484-8af64b5f-b3d3-4c08-b50e-4f4cf7654451.png)
i have looked at the documentation and im not sure whats wrong with this any help whould be great.
![Screenshot 2023-01-26 at 16 27 53](https://user-images.githubusercontent.com/89133704/214892185-725fd085-edf5-4075-82a6-55465bec7a9e.png)

**Comment by deejay-hub** [comment, marked as answer]
The Child Relationship name is **Related_Cases**. So the Relationship Name field in the metadata object should be **Related_Cases__r**. Try that and let us know if it solves your issue.

Looks like you have a custom object called AssistanceIncident__c. Then you have a child relationship to Cases via a custom field called Incident__c. Here you chose the relationship name Related_Cases. This value drives the relationship name Salesforce uses in SOQL queries.

**Reply by phillbarr** [reply]
thank you
